products/paas/repository.md

```diff
@@ -1,46 +1,22 @@
 # Repository
 
-The source code of your project will reside in a git-based VCS repository. You can start with a plain project, however we suggest starting with a fork of the [shopware/paas](https://github.com/shopware/paas) setup template from GitHub. You will find out more about the setup template in the [Setup Template](setup-template.md) section.
+The source code of your project will reside in a git-based VCS repository. You can start with a plain project, however we suggest starting with a new composer create-project. You will find out more about the setup template in the [Setup Template](setup-template.md) section.
 
 {% hint style="info" %}
 This guide explains the repository setup using **GitHub**. You can also integrate Bitbucket or GitLab based version control environments with Shopware PaaS. More information in [Source Integrations](https://docs.platform.sh/integrations/source.html).
 {% endhint %}
 
-## Fork and clone setup template
+## Create a Shopware project
 
-First of all create a new fork of the [shopware/paas](https://github.com/shopware/paas) setup template.
+First we create a new project with `composer create-project shopware/platform:dev-flex <folder-name>` using the [flex template](../../guides/installation/flex.md).
 
-![Screenshot of the fork button on GitHub](../../.gitbook/assets/fork-repository.png)
+This will create a brand new Shopware 6 project in the given folder. Now we have to change into our new created project and require the PaaS configuration with `composer req paas`.
```

Comment:
```suggestion
This will create a brand new Shopware 6 project in the given folder. Now, change it into the newly created project and require the PaaS configuration with `composer req paas`.
```

What do you mean by "require the PaaS configuration"? Is it written right?